packages/clay-form/stories/index.tsx

```diff
@@ -53,6 +56,90 @@ storiesOf('Components|ClayCheckbox', module)
 		</ClayCheckbox>
 	));
 
+storiesOf('Components|ClayForm', module)
```

Comment:
Ideally add different things to the Storybook otherwise people will look at the same information and we will have a redundancy of information.